### PR TITLE
Adds generic bank transfer icon

### PR DESCRIPTION
**Pull request by captn3m0**
As discussed on #12

### CODE REVIEW

File: app/assets/images/payment_icons/netbanking.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24"><title>bank</title><rect x="0.5" y="0.5" width="37" height="23" rx="3" ry="3" style="fill:#fff;stroke:#000;stroke-opacity:0.07000000029802322"/><polygon points="19 4.5 11.5 10.13 26.5 10.13 19 4.5"/><path d="M25.56,17.63H12.44a0.94,0.94,0,0,0-.94.94V19.5h15V18.56A0.94,0.94,0,0,0,25.56,17.63Z"/><rect x="19.94" y="11.06" width="1.88" height="5.63"/><rect x="23.69" y="11.06" width="1.88" height="5.63"/><rect x="16.19" y="11.06" width="1.88" height="5.63"/><rect x="12.44" y="11.06" width="1.88" height="5.63"/></svg>
```
Comment (andrewpaliga) [inline app/assets/images/payment_icons/netbanking.svg]: Can you start a new line here?

File: db/payment_icons.yml [hunk 2]
```diff
@@ -97,4 +97,8 @@
 -
   name: payzapp
   label: PayZapp
-  group: wallets
\ No newline at end of file
+  group: wallets
+-
+  name: netbanking
+  label: NetBanking
+  group: bank_transfers
```
Comment (andrewpaliga) [inline db/payment_icons.yml]: here as well.

File: app/assets/images/payment_icons/netbanking.svg (hunk 1 above)
Comment (andrewpaliga) [inline app/assets/images/payment_icons/netbanking.svg]: Is it possible to improve the resolution of the image at all? The bottom part looks a bit out of focus.

**Comment by captn3m0** [comment]
Will fix the newlines. Meanwhile, tagging @vikalp02 for the image resolution

**Comment by vikalp02** [comment]
@andrewpaliga It's an SVG icon so is vector based. Can you explain the issue with the bottom line in bank icon which I can improve upon?

**Comment by andrewpaliga** [comment]
@vikalp02 alright I think we're good for this PR then. Once the commit for the newlines is in I'll merge.

**Comment by captn3m0** [comment]
Fixed the newlines. 

Any timeline on when these merges will be pushed to production. I am still not seeing the new icons we pushed in #11 in our gateway configuration.

**Comment by captn3m0** [comment]
@andrewpaliga newlines added. Please merge :+1:

**Pull request merged**